packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -76,54 +66,6 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 
 	const { undo } = useDispatch( editorStore );
 
-	// Temporary home - should this live in `core-data`?
-	const fetchRichUrlData = useCallback(
```

Comment:
This intermediary function is no longer required as we will be allowing rich previews for internal URLs as well as external.